style.css

```diff
@@ -274,3 +284,142 @@ textarea {
   text-decoration: none;
   color: var(--dark);
 }
+
+/* Desktop Styles min width 720px */
+@media screen and (min-width: 768px) {
+  .nav__logo,
+  .nav__hamburger {
+    display: none;
+  }
+
+  .nav__links {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .nav__links-items {
+    list-style: none;
+    display: flex;
+  }
+
+  .nav__links-items a {
+    color: var(--dark);
+    font-weight: bold;
+  }
+
+  header {
+    background-image: url(./images/header-illustration-desktop.svg);
```

Comment:
- [ ] Currently, the background images are not getting displayed. I don't think the images are uploaded to the repository. Please look into the issue and update it accordingly. 🧐️

**Your Design**

![Screenshot from 2023-01-20 09-03-41](https://user-images.githubusercontent.com/17243325/213630223-b30256bd-b39a-4c9e-854c-3bb447a77f52.png)